pkg/util/ioutil/ioutil_test.go

```diff
@@ -3,166 +3,3 @@
 // license that can be found in the LICENSE file.
 
 package ioutil
```

Comment:
maybe remove this file entirely since there are no tests here?

Reply:
Done.